src/katsdpcontroller/schemas/product_config.json.j2

```diff
@@ -611,6 +611,13 @@
             "additionalProperties": false,
             "properties": {
                 "develop": {"type": "boolean", "default": false},
+{% if version >= "3.2" %}
+		"develop_opts":{
+                    "any_gpu": {"type": "boolean", "default": true},
```

Comment:
The properties need to go into a "properties" section. I'm a little surprised this is even compiling since "any_gpu" isn't a valid constraint (but then I think json-schema is somewhat permissive).

Also use an `additionalProperties: false` so that users can't add their own developer options that aren't already defined.